src/app/shared/card-content/card-content.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+import { NgClass } from '@angular/common';
+
+@Component({
+    selector: 'ngx-mask-demo-card-content',
+    templateUrl: './card-content.component.html',
+    styleUrls: ['./card-content.component.scss'],
+    standalone: true,
+    imports: [NgClass],
+})
+export class CardContentComponent {
+    @Input() public color!: string;
+    @Input() public title!: string;
+    @Input() public value!: string;
```

Comment:
If this component can not be without any field, add this field to selector
For example: title and value are required, then selector will be 'ngx-mask-demo-card-content[title][value]'